client/post-editor/media-modal/edit/edit-toolbar.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+var React = require( 'react' ),
+	noop = require( 'lodash/noop' );
+
+/**
+ * Internal dependencies
+ */
+var Gridicon = require( 'components/gridicon' );
+
+module.exports = React.createClass( {
+	displayName: 'MediaModalDetailEditToolbar',
+
+	propTypes: {
+		rotate: React.PropTypes.number,
+		scaleX: React.PropTypes.number,
+		scaleY: React.PropTypes.number,
+		setImageEditorState: React.PropTypes.func
+	},
+
+	getDefaultProps: function () {
+		return {
+			setImageEditorState: noop
+		};
+	},
+
+	rotate: function () {
+		var rotate = ( this.props.rotate - 90 ) % 360;
```

Comment:
`var` shouldn't be used in new code ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#variable-declarations))